src/sentry/snuba/metrics/extraction.py

```diff
@@ -591,16 +603,17 @@ def failure_tag_spec(_1: Project, _2: Optional[str]) -> List[TagSpec]:
     ]
 
 
-def apdex_tag_spec(project: Project, argument: Optional[str]) -> List[TagSpec]:
+def apdex_tag_spec(project: Project, arguments: Optional[List[str]]) -> List[TagSpec]:
     _, metric = _get_apdex_project_transaction_threshold(project)
 
     # TODO: we can also opt to fallback on the db threshold in case it's not supplied, but we have to see if we want to
     #  support that.
-    if argument is None:
+    if arguments is None:
         raise Exception("apdex requires a threshold parameter.")
 
     field = _map_field_name(metric)
-    apdex_threshold = int(argument)
+    (threshold,) = arguments
```

Comment:
This code will fail if the list has more than 1 element or if the list is empty. Could we just be more defensive and always take the first parameter independently iff the list is non-empty?

Reply:
```suggestion
    threshold = arguments[0]
```

Or you could do:

```suggestion
    (threshold, *_) = arguments
```

Reply:
What do you think?

Reply:
I have the same but with Rust 😅